ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -378,7 +378,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "uploadDocuments",
-    "Name": "Documents",
+    "Name": "Upload documents",
     "Description": "Upload documents",
     "DisplayOrder": 14,
     "PreConditionState(s)": "Open;Submitted;Gatekeeping;PREPARE_FOR_HEARING;RETURNED",
```

Comment:
Effects other states unfortunately. Need to catch up with UX as to wether this change is fine. Otherwise we may need to introduce a clone of this event.

Reply:
Agreed with Collette to rename documents event across all states.